tests/begin_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2015 The appc Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"io/ioutil"
+	"path"
+	"reflect"
+	"testing"
+
+	"github.com/appc/acbuild/Godeps/_workspace/src/github.com/appc/spec/aci"
+	"github.com/appc/acbuild/Godeps/_workspace/src/github.com/appc/spec/schema"
+	"github.com/appc/acbuild/Godeps/_workspace/src/github.com/appc/spec/schema/types"
+)
+
+func TestBegin(t *testing.T) {
+	tmpdir, err := setupTest()
+	if err != nil {
+		t.Fatalf("%v", err)
+	}
+
+	files, err := ioutil.ReadDir(path.Join(tmpdir, ".acbuild", "currentaci", aci.RootfsDir))
+	if err != nil {
+		t.Errorf("%v", err)
+	}
+	if len(files) != 0 {
+		t.Errorf("rootfs in aci contains files, should be empty")
+	}
+
+	manblob, err := ioutil.ReadFile(path.Join(tmpdir, ".acbuild", "currentaci", aci.ManifestFile))
+	if err != nil {
+		t.Errorf("%v", err)
+	}
+
+	man := &schema.ImageManifest{}
+
+	err = man.UnmarshalJSON(manblob)
+	if err != nil {
+		t.Errorf("invalid manifest schema: %v", err)
+	}
+
+	if g := man.Labels; !reflect.DeepEqual(g, make(types.Labels, 0)) {
+		t.Errorf("bad labels: expected no labels but got %+v", g)
+	}
+	if man.App != nil {
+		t.Errorf("manifest contains an app, shouldn't")
+	}
+	if len(man.Annotations) != 0 {
+		t.Errorf("manifest contains annotations, shouldn't")
+	}
+	if len(man.Dependencies) != 0 {
+		t.Errorf("manifest contains dependencies, shouldn't")
+	}
+	if len(man.PathWhitelist) != 0 {
+		t.Errorf("manifest contains a path whitelist, shouldn't")
+	}
+
+	err = cleanupTest(tmpdir)
+	if err != nil {
+		t.Errorf("%v", err)
```

Comment:
a) my super grammar nit, setupTest --> setUpTest and cleanupTest --> cleanUpTest
b) defer cleanUpTest() after setUpTest()
c) unless you have a good reason to care about the error that's actually related to acbuild behaviour, then don't log it using t.Errorf; either ignore or panic. any call to Errorf should reflect an issue with the actual lib behaviour, but AFAICT here you're just trying to be a good citizen?